testsuite/features/support/constants.rb

```diff
@@ -794,6 +795,8 @@
         sle-product-sles15-sp4-ltss-updates-x86_64
         sle-manager-tools15-pool-x86_64-sp4
         sle-manager-tools15-updates-x86_64-sp4
+        sle-manager-tools15-beta-pool-x86_64-sp4
+        sle-manager-tools15-beta-updates-x86_64-sp4
```

Comment:
Did you check if we need sync both?
```
        sle-manager-tools15-pool-x86_64-sp4
        sle-manager-tools15-updates-x86_64-sp4
```
+
```
        sle-manager-tools15-beta-pool-x86_64-sp4
        sle-manager-tools15-beta-updates-x86_64-sp4
```
?

Reply:
No, I did not check it. I assumed it is OK to have the beta ones on top of the normal ones. For HEAD it could be indeed interesting, but overall there are not that many packages.

HEAD
![image](https://github.com/uyuni-project/uyuni/assets/12104291/27fe3098-a478-41d1-9caa-6429fa568d28)

Uyuni
![image](https://github.com/uyuni-project/uyuni/assets/12104291/097c022c-f4b8-402d-b734-331336b38395)